packages/ia-components/sandbox/details/DetailsDownloadOptions.jsx

```diff
@@ -71,26 +77,31 @@ export default class DetailsDownloadOptions extends React.Component {
             >
               {formats('format', k).downloadable}
               {' '}
-              <I18nIcon className="iconochive-download" en="download"/>
+              <I18nIcon className="iconochive-download" en="download" />
             </AnchorDownload>
+            {(typeof DwebArchive === 'undefined' || !DwebArchive.mirror || (k !== 'Epub')) ? null : (
```

Comment:
what is this for? can this be abstracted out to a function so we can gain quick insight when we read through/use this file?

Reply:
Its the condition for why the nested HTML should be included. 
Sure it could be extracted out (and if it was more than one line I would do) but that would put the test (only used once) in a different place of the code than it use which I think would make it less readable. 
```
function isNotEpubOnMirror(format) { 
  return !DwebArchive && !DwebArchive.mirror || (format !== 'Epub')
}
```
But it still has to be used in that form since I believereact requires "null" if something is to be skipped. 
```
{ isNotEpubOnMirror(k) ? null : (
```

Reply:
would you be able to put it in a variable as a tldr?

Reply:
Please give me a code snipped of what you are thinking, everything I can think of as alternatives is WORSE (More confusing, more error prone) than doing this test in-line at the place its used.